### PR TITLE
Simplify boolean expressions with an simpler and equivalent.

### PR DESCRIPTION
**Pull request by arturobernalg**
* Simplify boolean expression with equivalent.

### CODE REVIEW

File: commons-numbers-combinatorics/src/main/java/org/apache/commons/numbers/combinatorics/FactorialDouble.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
      *
      * @param numValues Number of values of the function to compute.
      * @param cache Cached values.
-     * @throw IllegalArgumentException if {@code n < 0}.
+     * @throw CombinatoricsException if {@code n < 0}.
```
Comment (aherbert) [inline commons-numbers-combinatorics/src/main/java/org/apache/commons/numbers/combinatorics/FactorialDouble.java:41]: `CombinatoricsException` is package private; it should not be exposed in the javadoc. It extends IllegalArgumentException so the javadoc is still valid.

File: commons-numbers-primes/src/main/java/org/apache/commons/numbers/primes/SmallPrimes.java [hunk 2]
```diff
@@ -108,11 +108,11 @@ equivalence classes will have to hold (2-1)*(3-1)*(5-1)*(7-1)*(11-1) = 480
         of all integers.
          */
         final Set<Integer> primeNumbers = new HashSet<>();
-        primeNumbers.add(Integer.valueOf(2));
-        primeNumbers.add(Integer.valueOf(3));
-        primeNumbers.add(Integer.valueOf(5));
-        primeNumbers.add(Integer.valueOf(7));
-        primeNumbers.add(Integer.valueOf(11));
+        primeNumbers.add(2);
```
Comment (aherbert) [inline commons-numbers-primes/src/main/java/org/apache/commons/numbers/primes/SmallPrimes.java:111]: The current code makes the boxing explicit. Please avoid auto-boxing.

Comment (arturobernalg) [reply commons-numbers-primes/src/main/java/org/apache/commons/numbers/primes/SmallPrimes.java]: Changed

File: commons-numbers-combinatorics/src/main/java/org/apache/commons/numbers/combinatorics/FactorialDouble.java (hunk 1 above)
Comment (arturobernalg) [reply commons-numbers-combinatorics/src/main/java/org/apache/commons/numbers/combinatorics/FactorialDouble.java]: Changed

File: commons-numbers-combinatorics/src/main/java/org/apache/commons/numbers/combinatorics/LogFactorial.java [hunk 3]
```diff
@@ -41,7 +41,7 @@
      *
      * @param numValues Number of values of the function to compute.
      * @param cache Cached values.
-     * @throw IllegalArgumentException if {@code n < 0}.
+     * @throw CombinatoricsException if {@code n < 0}.
```
Comment (aherbert) [inline commons-numbers-combinatorics/src/main/java/org/apache/commons/numbers/combinatorics/LogFactorial.java:44]: You have left this `CombinatoricsException` exposed in the API.

Comment (arturobernalg) [reply commons-numbers-combinatorics/src/main/java/org/apache/commons/numbers/combinatorics/LogFactorial.java]: Changed
TY

**Comment by aherbert** [comment]
This is now only simplifying boolean expressions. Can you update the commit message to reflect that and then correct the formatting as per #103

**Comment by arturobernalg** [comment]
> This is now only simplifying boolean expressions. Can you update the commit message to reflect that and then correct the formatting as per #103

HI @aherbert 
Done
TY

**Comment by aherbert** [comment]
You changed the commit message but did not fix formatting to change 8 spaces to 4.

**Pull request closed without merging**